### PR TITLE
deprecated warning message

### PR DESCRIPTION
**Pull request by rotsuya**
add deprecated warning message.

### CODE REVIEW

**Comment by rotsuya** [comment]
How to test

1. `npm install` then `node build.js`
2. open 'index.html' via local web server
3. confirm no regression, comparing with https://nttcom.github.io/skyway/docs/ .
4. review text, links and design.

File: header.html [hunk 1]
```diff
@@ -11,6 +11,48 @@
 </head>
 
 <body>
+<div id="warn">
+    <div>
+        <h1>Deprecated!</h1>
+        <p>SkyWay is terminated as of March, 2018. Customers who are using SkyWay are requested to migrate to the succeeding service <a href="https://webrtc.ecl.ntt.com/en/?origin=skyway">ECLWebRTC</a>.<br>
```
Comment (alanmshelly) [inline header.html]:
```
We have released a new WebRTC platform, <a href="https://webrtc.ecl.ntt.com/en/?origin=skyway">ECLWebRTC</a>, to take the place of SkyWay. We will be shutting down the SkyWay servers in March 2018. Customers who are currently using SkyWay are required to migrate to ECLWebRTC by then or their services will stop working. If you are looking for the ECLWebRTC API reference, please see the <a href="https://webrtc.ecl.ntt.com/en/developer.html#sdk">ECLWebRTC API reference</a> page.
```

Comment (alanmshelly) [reply header.html]:
```
We have released a new WebRTC platform, <a href="https://webrtc.ecl.ntt.com/en/?origin=skyway">ECLWebRTC</a>, to take the place of SkyWay. We will be shutting down the SkyWay servers in March 2018. Customers who are currently using SkyWay are required to migrate to ECLWebRTC by then. Click <a href="https://webrtc.ecl.ntt.com/en/developer.html#sdk">here</a> for the ECLWebRTC API reference.
```

Comment (rotsuya) [reply header.html]: ちょうどいいです！

Comment (rotsuya) [reply header.html]: pushed again. PLZ review!

**Comment by alanmshelly** [comment]
LGTM

**Pull request merged**